style.md

```diff
@@ -1042,6 +1043,136 @@ func TestSigner(t *testing.T) {
 </td></tr>
 </tbody></table>
 
+### Embedding considered harmful
+
+Avoid embedding types in public structs.
+These obscure documentation and leak a detail about your implementation that
```

Comment:
nit: Leaking details >> inhibiting type evolution > obscured documentation, IMO - so maybe we can rewrite this as:

> Embedding can leak implementation details, make type evolution difficult, or obscure type documentation.

(or something along those lines)